### ISSUE TITLE
Obtain ESV certificate through esvp server

### ISSUE THREAD

**Issue by locksmithone** [usnistgov/ESV-Server#39]
I would like to propose this feature. Once the certificate is issued, an esvp client could obtain it from the server by providing eaID, test session, or similar.

**Comment by celic** [comment]
At the moment we can't do this. The certificate is created via a different system and that information never makes it back to something the API can access. I do see the use case here, and this is something we can work on, but it will take some time.

**Comment by celic** [comment]
We are working on this for the v1.6 release, likely Mar/Apr time frame.

**Comment by locksmithone** [comment]
Was this implemented? The protocol README is not updated to see how to interact with the server to request the certificate.

**Comment by celic** [comment]
Yes. GET /entropyCertificate/E# where # is the number you see on CSRC. It
may be entropyCertificates in the URL need to double check and add to the
document

**Comment by celic** [comment]
https://github.com/usnistgov/ESV-Server/blob/main/Entropy%20Source%20Validation%20Protocol.md#54-getting-entropy-certificate-information

It's not a ton of useful information. The `certificateId` returned can be used to construct a URL for `https://csrc.nist.gov/projects/cryptographic-module-validation-program/entropy-validations/certificate/1` where the last digit is the `certificateId`. As far as I'm aware, this should always match the `certificateNumber` though that may not be the case forever.
